Formula/riff.rb

```diff
@@ -0,0 +1,30 @@
+class Riff < Formula
+  desc "Pivotal open-source implementation of a Function As A Service"
+  homepage "https://www.projectriff.io/"
+  url "https://github.com/projectriff/riff.git",
+      :tag      => "v0.2.0",
+      :revision => "1ae190ff3c7edf4b375ee935f746ebfd1d8eaf5c"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    contents = Dir["{*}"]
+    (buildpath/"src/github.com/projectriff/riff").install contents
+    (buildpath/"bin").mkpath
+
+    cd "src/github.com/projectriff/riff" do
+      system "make"
+      bin.install "riff"
+    end
+  end
+
+  test do
+    expected_output = <<~EOS
```

Comment:
We need a test that exercises the some of the functionality of the app.

Reply:
That would require an external Kubernetes cluster.

Reply:
@fxcoudert is the test OK? I'm not sure I can do much more.

Reply:
You could check for a failure instead, maybe something that always returns "unautorized?"

Reply:
Done. I'm calling a command with an invalid kube config file.